custom/icds_reports/migrations/0008_20170301.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+from corehq.sql_db.operations import RawSQLMigration
+
+migrator = RawSQLMigration(('custom', 'icds_reports', 'migrations', 'sql_templates'))
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('icds_reports', '0007_20170215'),
+    ]
+
+    operations = [
+        migrator.get_migration('create_functions.sql'),
+    ]
```

Comment:
nit: i typically prefer giving migrations names rather than relying on the default django date/time